// @sapper/internal/manifest-server.mjs
// This file is generated by Sapper — do not edit it!
import component_0 from "../../../routes/index.svelte";
import component_1, { preload as preload_1 } from "../../../routes/blog/[permalink].svelte";
import component_2, { preload as preload_2 } from "../../../routes/tag/[tag].svelte";
import root from "../../../routes/_layout.svelte";
import error from "../../../routes/_error.svelte";

const d = decodeURIComponent;

export const manifest = {
	server_routes: [
		
	],

	pages: [
		{
			// index.svelte
			pattern: /^\/$/,
			parts: [
				{ name: "index", file: "index.svelte", component: component_0 }
			]
		},

		{
			// blog/[permalink].svelte
			pattern: /^\/blog\/([^\/]+?)\/?$/,
			parts: [
				null,
				{ name: "blog_$permalink", file: "blog/[permalink].svelte", component: component_1, preload: preload_1, params: match => ({ permalink: d(match[1]) }) }
			]
		},

		{
			// tag/[tag].svelte
			pattern: /^\/tag\/([^\/]+?)\/?$/,
			parts: [
				null,
				{ name: "tag_$tag", file: "tag/[tag].svelte", component: component_2, preload: preload_2, params: match => ({ tag: d(match[1]) }) }
			]
		}
	],

	root,
	root_preload: () => {},
	error
};

export const build_dir = "__sapper__/dev";

export const src_dir = "src";

export const dev = true;
