// @sapper/internal/manifest-client.mjs
// This file is generated by Sapper — do not edit it!
export { default as Root } from '../../../routes/_layout.svelte';
export { preload as root_preload } from './shared';
export { default as ErrorComponent } from '../../../routes/_error.svelte';

export const ignore = [];

export const components = [
	{
		js: () => import("../../../routes/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:index.svelte__"
	},
	{
		js: () => import("../../../routes/blog/[permalink].svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:blog/[permalink].svelte__"
	},
	{
		js: () => import("../../../routes/tag/[tag].svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:tag/[tag].svelte__"
	}
];

export const routes = (d => [
	{
		// index.svelte
		pattern: /^\/$/,
		parts: [
			{ i: 0 }
		]
	},

	{
		// blog/[permalink].svelte
		pattern: /^\/blog\/([^\/]+?)\/?$/,
		parts: [
			null,
			{ i: 1, params: match => ({ permalink: d(match[1]) }) }
		]
	},

	{
		// tag/[tag].svelte
		pattern: /^\/tag\/([^\/]+?)\/?$/,
		parts: [
			null,
			{ i: 2, params: match => ({ tag: d(match[1]) }) }
		]
	}
])(decodeURIComponent);

if (typeof window !== 'undefined') {
	import("/Users/asawo/blog/node_modules/sapper/sapper-dev-client.js").then(client => {
		client.connect(10000);
	});
}
